.travis.yml

```diff
@@ -12,7 +12,7 @@ before_script:
 - sleep 3
 script:
 - cd info.limpet.targetplatforms
-- mvn clean install
```

Comment:
The site target creates maven reports defined by the reporting element in pom.xml.
Sometime it will not work standalone (mvn site), so you should call "mvn clean verify site"